ISISComputingGroup/EPICS-ioc
# Ticket 2381: add records for showing valid frequencies

### Description of work

https://github.com/ISISComputingGroup/IBEX/issues/2381

### To test

I have added PREC to the frequency fields so 16,67 Hz doesn't appear as 17 Hz in the OPI.
Confirm this using the IOC in simulation mode.
```
caput %MYPVPREFIX%MK2CHOPR_01:FREQ:SP 16.67
```

I have added a mbbo record that is used to set the FREQ:SP to "hard-coded" values only via the OPI. This requires the OPI changes also associated with this ticket.

### Acceptance criteria
The above tests work.

---

#### Code Review


### Functional Tests


### Final steps
